src/Command/AnalyzeCommand.php

```diff
@@ -81,7 +81,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         new ConsoleFormatter($this->dispatcher, $output);
 
         $parser = new NikicPhpParser();
-        $astMap = $this->astRunner->createAstMapByFiles($parser, $this->dispatcher, $this->collectFiles($configuration));
+        $astMap = $this->astRunner->createAstMapByFiles($parser, $this->collectFiles($configuration));
```

Comment:
this seems to break the `createAstMapByFiles()` method's interface: https://github.com/sensiolabs-de/astrunner/blob/master/src/AstRunner.php#L21-L25